Content/default/webpack.config.js

```diff
@@ -80,24 +86,24 @@ module.exports = {
     plugins: isProduction ?
         commonPlugins.concat([
             new MiniCssExtractPlugin({ filename: 'style.[name].[hash].css' }),
-            new CopyWebpackPlugin({ patterns: [{ from: resolve(CONFIG.assetsDir) }]}),
+            new CopyWebpackPlugin({ patterns: [{ from: resolve(CONFIG.assetsDir) }] }),
         ])
-        : commonPlugins.concat([
-            new webpack.HotModuleReplacementPlugin(),
```

Comment:
are we removing support for HMR?

Reply:
Now it's added by default by webpack when you add `hot: true` to the devServer, check the docs  https://webpack.js.org/configuration/dev-server/#devserverhot there is a tip that details this.